packages/FormElement/src/FormElement.js

```diff
@@ -132,6 +132,24 @@ function FormElement(props) {
     "aria-invalid": hasError,
   };
 
+  const componentNameMap = {
+    DATEPICKER: "DatePicker",
+    INPUT: "Input",
+    LISTBOX: "ListBox",
+  };
+
+  const supportedComponents = [componentNameMap.DATEPICKER, componentNameMap.INPUT, componentNameMap.LISTBOX];
```

Comment:
This part can be stored outside of the functional component?

Reply:
You mean th array? Yes it could be in some kind of helper and imported but for now I wanted to keep it simple until we other components need a similar lookup..